src/frog-base.tsx

```diff
@@ -675,15 +687,16 @@ export class FrogBase<
               <meta
                 property="fc:frame:image"
                 content={
-                  context.status === 'initial'
+                  context.status === 'initial' && this.initialImageMasking
                     ? `${context.url}/image`
                     : imageUrl
```

Comment:
What was the motivation behind this behavior again? I wonder if we can avoid it altogether so we don't have to do these workarounds which don't function well in other environments.

Reply:
As far as I understand by the code, and [this PR](https://github.com/wevm/frog/pull/222) the reason was to make the first frame dynamic. Say you have the frame:image metatag pointing to `myFrame.com/image` which is cached by warpcast. You can't put a static image in that route, so the dynamic image must be somewhere else so that when warpcast calls the /image path a function returns the image from the dynamic attribute in the meta tag. Btw I'm not sure this is not a bug from the warpcast team. the cache-control thing they did doesn't seem to work intuitively. Probably the best is reach out to them and find a clean solution.

Reply:
> What was the motivation behind this behavior again? I wonder if we can avoid it altogether so we don't have to do these workarounds which don't function well in other environments.

It was added to support refreshing of the first frame image.

There are basically two rules we have to comply to:
- image response must have a `Cache-Control` header;
- image url must be the same across "refreshes".

The latter was the focus, as before #222, the initial frame image url image always depended on the image itself (as it is compressed and put to query), which makes refreshing never possible.

`frog:image` was introduced to store the full url while `fc:frame:image` stays the same across updates.